### PR TITLE
tests: remove opensuse-15.1 and add opensuse-15.3 from spread runs

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Update the spread runs to support opensuse-15.3-64

The new image is already deployed to gce

Tests and packaging are updated accordingly

### CODE REVIEW

File: packaging/opensuse-15.1 [hunk 1]
```diff
@@ -1 +0,0 @@
-opensuse
```
Comment (anonymouse64) [inline packaging/opensuse-15.1:1]: I notice we still have packaging/opensuse-15.0, is that needed still since we are dropping 15.1 ?

Comment (anonymouse64) [reply packaging/opensuse-15.1:1]: Ah the git diff shows it as just moving from opensuse-15.0 to opensuse-15.3 sorry I was just confused

**Review by anonymouse64 — APPROVED** [review]
LGTM

**Comment by mvo5** [comment]
Nice!

**Pull request merged**